main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
+
+GENRES = ["Western", "Action", "Dramma"]
+ACTORS = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson")
+]
+
 
 def main() -> QuerySet:
-    pass
+    for genre in GENRES:
```

Comment:
Genres and Actors are used as global variables in your code which is not a good practice - you either declare them inside main or pass to main() as arguments